bin/configure.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+# This program and the accompanying materials are
+# made available under the terms of the Eclipse Public License v2.0 which accompanies
+# this distribution, and is available at https://www.eclipse.org/legal/epl-v20.html
+# 
+# SPDX-License-Identifier: EPL-2.0
+# 
+# Copyright Contributors to the Zowe Project.
+
+
+# Required variables on shell:
+# - ROOT_DIR
+# - WORKSPACE_DIR
+# - NODE_HOME
+if [ -n "$NODE_HOME" ]
+then
+  NODE_BIN=${NODE_HOME}/bin/node
+else
+  NODE_BIN=node
+fi
+
+cd ${ROOT_DIR}/components/app-server/share/zlux-app-server/bin
+. ./convert-env.sh
+
```

Comment:
Need to confirm if the nodejs env vars such as _TAG_REDIR_OUT=txt is set at this time, because otherwise it would need to be set explicitly here.